src/middleware-wrapper.js

```diff
@@ -12,17 +12,18 @@ const middlewareWrapper = (config) => {
     fs.readFileSync(path.join(__dirname, '/public/index.html'))
       .toString()
       .replace(/{{title}}/g, config.title)
+      .replace(/{{port}}/g, config.port)
```

Comment:
Port of socket.io server which can be "plugged" or bind to express server (accordingly to socket.io documentation)